src/Microsoft.Health.Fhir.TemplateManagement/Models/TemplateManagementErrorCode.cs

```diff
@@ -41,12 +41,12 @@ public enum TemplateManagementErrorCode
         // OverlayException
         ImageLayersNotFound = 3101,
         OverlayMetaJsonInvalid = 3102,
-        SequenceNumberReadFailed = 3103,
-        SortLayersFailed = 3104,
-        BaseLayerNotFound = 3105,
+        BaseLayerLoadFailed = 3103,
 
         // OrasException
         OrasTimeOut = 3201,
         OrasProcessFailed = 3202,
+        CacheManifestFailed = 3203,
+        CacheBlobFailed = 3204,
```

Comment:
It would be better to keep the `oras` prefix if these exceptions are only raised by oras, e.g., `OrasCacheBlobFailed`.

Reply:
Thanks, renamed it.